tests/ga/test_extension.py

```diff
@@ -1664,6 +1664,56 @@ def mock_popen(cmd, *args, **kwargs):
                                             expected_msg="Dependent Extension OSTCExtensions.OtherExampleHandlerLinux did not reach a terminal state within the allowed timeout. Last status was {0}".format(
                                                 ValidHandlerStatus.warning))
 
+    def test_it_should_not_create_placeholder_for_aks_extension(self, mock_http_get, mock_crypt_util, *args):
```

Comment:
Just a FYI, As a part of new PR which recently got merged all new test cases should be a part of the new TestExtension class. It was created to use the newer 'mock_wire_protocol' . Since this is a release branch this shouldn't be an issue.

Reply:
Thanks for the update (and also for creating that new class). 
Since this is on the release branch, it still doesn't have that new class in here so I just added it to the old class. I'll make sure to add new tests to the new class moving forward though :)